content/docs/configuration/venafi.md

```diff
@@ -224,6 +229,13 @@ Once the Secret containing credentials has been created, you can create your
 resource, you must change the `kind` field to `ClusterIssuer` and remove the
 `metadata.namespace` field.
 
+> Note: If you are configuring your issuer as a `ClusterIssuer` resource in
+> order to issue `Certificates` across your whole cluster, you must set the
+> `--namespace` parameter to `cert-manager`, which is the default `Cluster
+> Resource Namespace`. The `Cluster Resource Namespace` can be configured
+> through the `--cluster-resource-namespace` flag on the cert-manager controller
+> component.
```

Comment:
```suggestion
> ℹ️  If you are using a `ClusterIssuer` resource, the Secret containing the credentials must be in the `Cluster
> Resource Namespace`, which is `cert-manager` by default. The `Cluster Resource Namespace` can be configured
> through the `--cluster-resource-namespace` flag on the cert-manager controller
> component.
> 
> 📖 Read [Issuer Configuration](./README.md#cluster-resource-namespace) to learn more about the concept of a ClusterIssuer and the Cluster Resource Namespace
```

I wanted to make it clear that the **Secret** must be in the cert-manager namespace.
And there's a good explanation of this in https://cert-manager.io/docs/configuration/